src/views/clusteroverview/OverviewTab/metric-charts-card/utils/hooks/useXAxisTicks.ts

```diff
@@ -0,0 +1,66 @@
+import { dateFormatterNoYear } from '@kubevirt-utils/components/Timestamp/utils/datetime';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+
+import { getLabeledTickIndexes } from '../utils';
+
+import { UseXAxisTicks } from './types';
+import { getPointDay } from './utils';
+
+// Ticks should be placed at the center point of each day's data
+const useXAxisTicks: UseXAxisTicks = (chartData) => {
+  const { t } = useKubevirtTranslation();
+  if (!chartData) {
+    return [null, null];
+  }
+
+  const getPrevDay = (pointDay: number): number => (pointDay === 0 ? 6 : pointDay - 1);
```

Comment:
maybe constants with nameing? i find it difficult to me to understand what is point data ,0 or 6

Reply:
Fixed!